content/about/notes/1.1/index.md

```diff
@@ -18,15 +18,50 @@ TODO: Need to have an explicit section on everything that's being deprecated.
 
 ## Upgrades
 
-- **Helm Changes**.
-TBD: need full content & links
-
-- things to watch out for/configure before you upgrade - from all subsystems below. For example, outbound traffic is allowed by default for unknown ports, what else?
-TBD: need full content & links
-
-    - Egress gateway is disabled by default
-    - Mixer policy is off by default
-    - Outbound traffic policy is set to `ALLOW_ANY`. Traffic to unknown ports will be forwarded as-is.
+- **New `istio-init` Helm Chart**. Istio’s CRDs have been placed into their own Helm chart `istio-init`.
+  This preserves custom resource data when using `helm install`, facilitates the upgrade process,
+  and enables the Istio project to evolve beyond a Helm-based installation.
+- **Improved upgrade approach**.  The [upgrade procedure](/docs/setup/kubernetes/upgrade/) provides
+  the proper procedure for upgrading from Istio 1.0.6 to Istio 1.1.0.  If upgrading, please follow instructions
+  **carefully and precisely**.  If using Istio's built-in certmanager implementation, you must use the
+  `--set certmanager=true` flag when installing both istio-init and istio charts with either
+  `template` or `tiller` installation modes.
+- **Improved Multicluster Integration**. The 1.0 `istio-remote` chart used for
+  [multicluster VPN](/docs/setup/kubernetes/install/multicluster/vpn/) and
+  [multicluster split horizon](/docs/examples/multicluster/split-horizon-eds/) remote cluster installation
+  has been consolidated into the Istio chart.  This simplies the operator experience. To generate a 1.0
+  equivalent `istio-remote` chart, use the flag `--set global.istioRemote=true`.
+- **Reduced load balancer requirements for addons**.  Addons are no longer exposed via separate load balancers.
+  Instead addons are exposed via the ingressgateway.  To expose an addon via the ingressgateway,
+  please use the [Addon Gateway documentation](/docs/tasks/telemetry/gateways/).
+  -**More flexibility with statsd collector**.  The built-in Istio statsd collector has been removed.
```

Comment:
what is the user action for this for upgrade?

Reply:
it would be good to categorize the list to actionable items vs FYI items.   And single cluster vs multiclusters.

Reply:
I don't know what the user action is for statsd.  It is actionable, there are flags, but statsd integration documentation is not present.

WIll reorg single ahead of multiple clusters once a few other reviewers get a crack at this PR.

Reply:
Did this happen in 1.1?  My memory is hazy, but I feel like this happened in the 1.0.x series.

Reply:
@douglas-reid it was an emergency fix in 1.0 - so some people have it and some dont...

Reply:
ignore my prior comment - i was reading the wrong thing.

statsd deprecation may have happend in a z stream, but it shipped ini 1.0 and some people may have statsd running and now don't after 1.1.